app/locker/data/dashboards/BaseDashboard.php

```diff
@@ -34,17 +37,8 @@ public function getGraphData(\DateTime $startDate = null, \DateTime $endDate = n
    *
    **/
   public function statementCount(){
-
-    $collection = \DB::getCollection('statements');
-
-    $query = [];
-    
-    //Limit by LRS
-    if( $this->has_lrs ){
-      $query['lrs_id'] = new \MongoId($this->lrs);
-    }
-
-    return $collection->count($query);
+    $lrs_id = $this->has_lrs ? $this->lrs : null;
```

Comment:
Sorry that wasn't quite correct, it is set in the constructor, but explicitly depending on the type of dashboard that extends the BaseDashboard (which should really be abstract). E.g the LrsDashboard declares itself as having an LRS...
https://github.com/LearningLocker/learninglocker/blob/issue/lrs-statement-counts/app/locker/data/dashboards/LrsDashboard.php#L11